bitnami/mariadb-galera/README.md

```diff
@@ -183,17 +186,23 @@ LDAP support can be enabled in the chart by specifying the `ldap.` parameters wh
 - `ldap.binddn`: LDAP bind DN. No defaults.
 - `ldap.bindpw`: LDAP bind password. No defaults.
 - `ldap.bslookup`: LDAP base lookup. No defaults.
+- `ldap.nss_initgroups_ignoreusers`: LDAP ignored users. `root,nslcd`.
+- `ldap.scope`: LDAP search scope. No defaults.
+- `ldap.tls_reqcert`: LDAP TLS check on server certificates. No defaults.
 
 For example:
 
 ```bash
 $ helm install --name my-release bitnami/mariadb-galera \
     --set ldap.enabled="true" \
-    --set ldap.url="ldap://my_ldap_server" \
-    --set ldap.base="dc=example,dc=org" \
-    --set ldap.binddn="cn=admin,dc=example,dc=org" \
+    --set ldap.uri="ldap://my_ldap_server" \
+    --set ldap.base="dc=example\,dc=org" \
```

Comment:
Why is the backslash necessary? AFAIK a comma is a valid character and does not need to be escaped.

Reply:
Removing backslashes